udata/templates/home.html

```diff
@@ -57,7 +57,7 @@ <h1 class="page-header">
 
                 <div class="row">
                     <ul class="card-list">
-                        {% for dataset in current_site.settings.home_datasets %}
+                        {% for dataset in current_site.settings.home_datasets|rejectattr('deleted') %}
```

Comment:
Mmmh the stock should be quite low and not contain that much removed datasets or am I wrong? Putting a piece of code to handle that temporary issue doesn't look worth it but I'm not sure I get the whole picture.